charts/operator-wandb/charts/parquet/templates/cron.yaml

```diff
@@ -0,0 +1,107 @@
+---
+{{- if .Values.backfillEnabled }}
+{{- $imageCfg := dict "global" $.Values.global.image "local" $.Values.image -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "parquet.fullname" . }}-backfill
+  labels:
+    {{- include "wandb.commonLabels" . | nindent 4 }}
+    {{- include "parquet.commonLabels" . | nindent 4 }}
+    {{- include "parquet.labels" . | nindent 4 }}
+    {{- if .Values.cronjob.labels -}}
+    {{-   toYaml .Values.cronjob.labels | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- if .Values.cronjob.annotations -}}
+    {{-   toYaml .Values.cronjob.annotations | nindent 4 }}
+    {{- end }}
+spec:
+  schedule: "* * * * *"
```

Comment:
I think we should only run backfill by default at a very slow pace. Something like once every hour should be good enough. For particular customers, we can go and update the frequency so we can ramp them out faster.

Alternate approach is to include a `BackfillConfig` like we discussed, but looks like `backfillEnabled` and `backfillCron` together should achieve the same affect, via helm, so it'll be easier to change.